renovate.json

```diff
@@ -207,6 +207,18 @@
       "minimumReleaseAge": "7 days",
       "enabled": true
     },
+    {
+      "groupName": "CodeQL",
+      "matchDepNames": [
+        "github/codeql-action/analyze",
+        "github/codeql-action/init"
+      ],
+      "reviewers": ["team:kibana-security"],
+      "matchBaseBranches": ["main"],
+      "labels": ["Team:Security", "release_note:skip", "backport:all-open"],
+      "minimumReleaseAge": "7 days",
```

Comment:
I pulled this from a different section owned by you all, can modify if there's a different preference.

Reply:
Perfect!